ocs_ci/deployment/helpers/external_cluster_helpers.py

```diff
@@ -378,7 +383,12 @@ def get_rgw_endpoint():
 
     """
     rgw_endpoint = None
+    zone = config.ENV_DATA.get("zone")
```

Comment:
@petr-balogh , I am not able to visualize about the source of this zone info. the data center zone info from config.EXTERNAL_MODE["external_cluster_node_roles"] comes from the external cluster yaml that we will have but I was wondering from where this ENV_DATA.zone gets populated? I tried dumping node info from a vmware cluster and I don't see any, if we are passing this zone info from a config file through --ocsci-conf, which file is that ? Am I missing something totally!

Reply:
This is populated from multicluster job from this place: https://gitlab.cee.redhat.com/ocs/ocs4-jenkins/-/blob/master/jobs/pipelines/qe_dr_setup/Jenkinsfile#L46 where we are creating the yaml text config.

This we looked at with @dahorak and should be created for each cluster.

Reply:
But for non mdr job it's not needed and I don't want to break it.

Reply:
@petr-balogh , got it, my mistake, I forgot that this is also related to single cluster as well.